<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctrlobj</class>
 <widget class="QWidget" name="ctrlobj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ctrlobj</string>
  </property>
  <widget class="QPushButton" name="bt_Connected">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_Baud">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>1000k</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <item>
    <property name="text">
     <string>1000k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>800k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>666k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>400k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>250k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>125k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50k</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>帧类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>帧格式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>帧ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>帧数据：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_EXT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>标准帧</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <item>
    <property name="text">
     <string>标准帧</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>扩展帧</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="CB_RTR">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>数据帧</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <item>
    <property name="text">
     <string>数据帧</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>远程帧</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BT_Send">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T_ID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00000008</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="T_DATA">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 01 02 03 04 05 06 07</string>
   </property>
   <property name="maxLength">
    <number>23</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
